### PR TITLE
Update German Translation

### PR DESCRIPTION
**Pull request by arch1t3cht**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -2575,15 +2460,15 @@ msgstr "Durchlaufe Modi der Tag-Anzeige"
 
 #: ../src/command/grid.cpp:266
 msgid "ASS Override Tag mode set to show full tags."
-msgstr "ass-Tag-Modus auf 'alle Tags' setzen."
+msgstr "Ass-Tag-Modus auf 'alle Tags' setzen."
```
Comment (FichteFoll) [inline po/de.po:2463]: "ASS"
(same for the following two strings)

File: po/de.po [hunk 2]
```diff
@@ -2812,30 +2692,24 @@ msgid "Add lead in and out"
 msgstr "Ein- und Auslaufbereiche hinzufügen"
 
 #: ../src/command/time.cpp:240
-#, fuzzy
-#| msgid "Create an audio clip of the selected line"
 msgid "Add both lead in and out to the selected lines"
-msgstr "Erzeuge einen Audioclip für die ausgewählte Zeile"
+msgstr "Sowohl Ein- als auch Auslaufbereiche zu markierten Zeilen hinzufügen"
```
Comment (FichteFoll) [inline po/de.po:2696]: `"Ein- und Auslaufbereiche zu markierten Zeilen hinzufügen"`
should suffice

File: po/de.po [hunk 3]
```diff
@@ -3795,34 +3634,30 @@ msgid ""
...
 msgid "TTXT import"
-msgstr "Stilimport"
+msgstr "TTXT-Import"
 
 #: ../src/preferences.cpp:118
-#, fuzzy
-#| msgid "style import"
 msgid "Plain text import"
-msgstr "Stilimport"
+msgstr "Reintext-Import"
```
Comment (FichteFoll) [inline po/de.po:3660]: "Klartext"

File: po/de.po [hunk 4]
```diff
@@ -4756,6 +4577,10 @@ msgid ""
 "\n"
 "Error Message: %s"
 msgstr ""
+"Entschuldigung, ein Fehler ist aufgetreten. Bitte speichern Sie Ihre Arbeit "
+"und starten Aegisub neu.\n"
```
Comment (FichteFoll) [inline po/de.po:4581]: …starten Sie Aegisub neu.

**Review by FichteFoll — COMMENTED** [review]
Took a quick glance at a couple of these strings and added a few suggestions. Rest LGTM.

File: po/de.po [hunk 5]
```diff
@@ -6649,34 +6426,33 @@ msgid ""
 "\n"
 "See the help file for information on how to use this."
 msgstr ""
+"Wendet Karaoke-Vorlage an.\n"
+"\n"
+"Siehe Hilfedata für näherer Infos zur Verwendung."
```
Comment (TheOneric) [inline po/de.po:6431]: Hilfedata → Hilfedatei

File: po/de.po [hunk 6]
```diff
@@ -6440,7 +6228,7 @@ msgstr "Konnte nichts vom Update-Server herunterladen."
 #: ../src/dialog_version_check.cpp:312
 #, c-format
 msgid "HTTP request failed, got HTTP response %d."
-msgstr "http-Anforderung fehlgeschlagen, bekam http-Antwort %d"
+msgstr "HTTP-Anfrage fehlgeschlagen mit Code %d fehlgeschlagen."
```
Comment (TheOneric) [inline po/de.po:6231]: `HTTP-Anfrage mit Code %d fehlgeschlagen.`

File: po/de.po [hunk 7]
```diff
@@ -5084,21 +4904,19 @@ msgid "All fonts found.\n"
 msgstr "Fertig. Alle Schriftarten gefunden.\n"
 
 #: ../src/font_file_lister.cpp:211
-#, fuzzy, c-format
-#| msgid "%d fonts could not be found.\n"
+#, c-format
 msgid "One font could not be found\n"
 msgid_plural "%d fonts could not be found.\n"
-msgstr[0] "%d Schriftarten konnten nicht gefunden werden.\n"
+msgstr[0] "Eine Schriftart konnte nicht gefunden worden.\n"
```
Comment (TheOneric) [inline po/de.po:4910]: worden  → werden

*(not sure how i managed to create this new error; maybe i initially reworded it to “Eine Schriftart wurde nicht gefunden.” but reverted it for consistency with the plural and typoed?)*

File: po/de.po [hunk 8]
```diff
@@ -2285,71 +2180,63 @@ msgstr "Einfügen des unübersetzten Textes"
...
 
 #: ../src/command/app.cpp:69
 msgid "&Audio+Subs View"
-msgstr "Audio+Untertitel anzeigen"
+msgstr "&Audio+Untertitel Ansicht"
 
 #: ../src/command/app.cpp:70
 msgid "Audio+Subs View"
-msgstr "Audio+Untertitel anzeigen"
+msgstr "Audio+Untertitel Anzeigen"
```
Comment (TheOneric) [inline po/de.po:2199]: For consistency with other strings: `Anzeigen` → `Ansicht`

**Review by TheOneric — COMMENTED** [review]
going through this again, here are some more minor things to fix

**Comment by arch1t3cht** [comment]
Thanks, all applied, see https://github.com/TypesettingTools/Aegisub/pull/213 if you want to check through it.

I'm pretty sure the worden/werden error is mine, not yours. Since all the different PRs were based on different .po files I manually went through your diff and updated TypesettingTools/Aegisub's current .po file (do let me know if this was stupid and there was a better way to do this, I'm not too experienced with the whole gettext ecosystem).

**Comment by TheOneric** [comment]
After the po is updated to the newest POT, you can load all matching entries from another translation file using `msgmerge`, then you only need to clean up some fuzzy matches

**Pull request merged**